namematch/generate_must_links.py

```diff
@@ -58,23 +58,22 @@ def main(self, **kw):
 
         # get records with non-missing unique identifiers
         ml_var_df = self.build_ml_var_df(
-                self.all_names_file, uid_vars_list, eid_vars_list)
+                self.all_names_file, uid_vars_list)
 
         # get the "must-link" record pairs
         must_links_df = self.get_must_links(
                 ml_var_df, uid_vars_list, eid_vars_list)
 
         # true record pairs
-        must_links_df.to_csv(
+        must_links_df.to_parquet(
             self.must_links,
-            index=False,
-            quoting=csv.QUOTE_NONNUMERIC
+            index=False

```

Comment:
Now we change the output must link file to parquet, we'll need to also change the default must link output file name to `must_link.parquet` instead of `must_link.csv` in `namematcher.py` to avoid confustion.

Reply:
Good catch! Committed that change.